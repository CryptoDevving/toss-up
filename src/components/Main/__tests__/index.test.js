import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import Main from '..';

afterEach(cleanup);

describe('Main component', () => {
  it('renders', () => {
    render(<Main />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Main />);

    expect(asFragment()).toMatchSnapshot();
  });

})