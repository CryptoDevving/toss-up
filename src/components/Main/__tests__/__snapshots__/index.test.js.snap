// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      style="transition: opacity 400ms, transform 400ms; transform: translateY(20px); opacity: 0;"
    >
      <div
        class="container-fluid mt-5 col-m-4"
        style="max-width: 550px;"
      >
        <div
          class="col-sm"
        >
          <main
            class="col-lg-12 text-monospace text-center text-light"
            role="main"
          >
            <div
              class="content mr-auto ml-auto"
            >
              <div
                class="mt-3"
                id="content"
              >
                <div
                  class="card mb-4"
                >
                  <div
                    class="card-body"
                  >
                    <div>
                      <img
                        alt="logo"
                        src="eth.png"
                        width="225"
                      />
                    </div>
                     
                    <p />
                    <div
                      class="input-group mb-4"
                    >
                      <input
                        class="form-control form-control-md"
                        placeholder="bet amount..."
                        required=""
                        step="0.01"
                        type="number"
                      />
                      <div
                        class="input-group-append"
                      >
                        <div
                          class="input-group-text"
                        >
                          <img
                            alt=""
                            height="20"
                            src="eth.png"
                          />
                           
                          <b>
                            ETH
                          </b>
                        </div>
                      </div>
                    </div>
                    <button
                      class="btn low btn-sm"
                      type="submit"
                    >
                      Low
                    </button>
                       
                    <button
                      class="btn high btn-sm"
                      type="submit"
                    >
                      High
                    </button>
                  </div>
                  <div
                    class="numbers"
                  >
                    <div
                      class="spinner-border fix float-right"
                      id="loader"
                      role="status"
                    />
                    <div
                      class="dice2"
                    >
                      <img
                        alt="logo"
                        src="dice_logo.png"
                        width="80"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </main>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
